src/AppBundle/Controller/IdeaController.php

```diff
@@ -30,9 +34,25 @@ public function viewAction($id)
         ));
     }
 
-    public function createIdeaAction()
+    /**
+     * @param Request $request
+     * @Route("/idea/create", name="idea_create")
```

Comment:
Why is this here? As far as I know, the routing annotations don't work? for reasons i forgot... There is a `routes.yml` in the bundle